source/routes.ts

```diff
@@ -28,12 +35,55 @@ export function findRouteMethodByType(methods: Method[], type: string) {
   throw new Error(`Method with type "${type}" not found`);
 }
 
+function cloneOverrides(overrides: MethodOverride[]) {
+  return overrides.map((override) => ({ ...override }));
```

Comment:
I could update the override name and add a `(global)` in the end like suggested in [this comment](https://github.com/rhberro/the-fake-backend/issues/32#issuecomment-625385798).
```ts
function cloneOverrides(overrides: MethodOverride[]): MethodOverride[] {
  return overrides.map((override) => ({
    ...override,
    name: `${override.name} (Global)`,
  }));
}
```

What do you think about this @fsmaia and @rhberro ?